modules/Chromecast/resources/sender/chromecast.js

```diff
@@ -69,12 +69,13 @@
          * @param reason
          */
         toggleCastButton: function ( isAvailable, reason ) {
+            var _this = this;
             this.log( "toggleCastButton: isAvailable=" + isAvailable + ", reason=" + reason );
             if ( isAvailable ) {
                 this.initializeCastApi();
-                this.show();
+                this.embedPlayer.playerReadyFlag ? this.show() : this.bind('playerReady', function (){_this.show()});
```

Comment:
I would prefer to keep it DRY, so create an inline function called loadWhenReady and call it with the callback function as a prameter.

Reply:
Done